cmd/util/api.go

```diff
@@ -34,13 +35,24 @@ func GetAPIClientV2(cmd *cobra.Command, cfg types.Bacalhau) (clientv2.API, error
 	}
 	base := fmt.Sprintf("%s://%s:%d", apiSheme, apiHost, apiPort)
 
+	metastore := repo.NewMetadataStore(cfg.DataDir)
+	installationID, err := metastore.ReadInstallationID()
+	if err != nil {
+		return nil, err
+	}
+	instanceID, err := metastore.ReadInstanceID()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
these should be best effort instead of failing the request

Reply:
Will modify to log a debug message, and only include in headers if found and non-empty.